README.md

```diff
@@ -10,4 +10,10 @@ Future plans including adding the ability to:
 * Analyze and diagnose common issues
 * Automate common cloud management and maintenance tasks
 
-Documentation is maintained [HERE](http://goldstone-server.readthedocs.org/en/latest/)
+Goldstone Server is licensed under the Apached 2.0 license (see LICENSE file)
+
+Goldstone Server Enterprise components are licensed under the Solinea Software License Agreement v1 (see GSE_LICENSE.pdf)
```

Comment:
If we put it in the submodule, it will get burried in the container's venv source tree.  Not a bad idea to have it in there, but this README gets mapped into the /opt/goldstone dir.  Thoughts?